portal/client/src/app/auth/signup/signup.component.ts

```diff
@@ -29,16 +30,20 @@ export class SignupComponent implements OnInit {
       return
     }
     if (event.form.value.password != event.form.value.passwordVerif) {
-        this.message = "your password is not the same than your password confirmation"
+        this.messageError = "your password is not the same than your password confirmation"
         return
     }
     this.httpService.signup(event.form.value.username, event.form.value.password, event.form.value.email).subscribe(
       data => {
-        this.message = "Your account is created, you are going to receive an email. Confirmed you received it, before sign in"
+        this.message = "Your account is created, you are going to receive an email to validate your account"
```

Comment:
Yes, and if IRC, we set an environment variable for this when we run the cluster create command. I'm commenting from my phone right now and can check later, but you can also check `cli\commands\cluster\create.go`